Core.ApplicationServices/Model/SystemUsage/Write/UpdatedSystemUsageGeneralProperties.cs

```diff
@@ -13,7 +13,7 @@ public class UpdatedSystemUsageGeneralProperties
         public OptionalValueChange<Maybe<Guid>> DataClassificationUuid { get; set; } = OptionalValueChange<Maybe<Guid>>.None;
         public OptionalValueChange<string> Notes { get; set; } = OptionalValueChange<string>.None;
         public OptionalValueChange<string> SystemVersion { get; set; } = OptionalValueChange<string>.None;
-        public OptionalValueChange<Maybe<(int lower, int? upperBound)>> NumberOfExpectedUsersInterval { get; set; } = OptionalValueChange<Maybe<(int lower, int? upperBound)>>.None;
+        public OptionalValueChange<Maybe<(int? lower, int? upperBound)>> NumberOfExpectedUsersInterval { get; set; } = OptionalValueChange<Maybe<(int? lower, int? upperBound)>>.None;
```

Comment:
we should leverage the "maybe" in stead since it's a maybe of the entire value tuple. That will also make interpretation simpler since Maybe.None will just become "UNDECIDED"